spec/facades/session_facade_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe SessionFacade do
+  it "should return session details for a user" do
+    user = SessionFacade.session_details(stub_omniauth)
+
```

Comment:
If you wanted to test the whole response structure without having a huge list of assertions, you could add it to the spec>helper.rb file. Then you can just call that within any tests that need to reference the response for user call. There's some examples in there on garden and sensor if you're curious.

Reply:
JKKKK you already did that.